src/components/node/stanford-person/libguide.tsx

```diff
@@ -11,45 +11,21 @@ type Props = HTMLAttributes<HTMLDivElement> & {
 }
 
 const LibGuides = ({guides, headingLevel = 2, ...props}: Props) => {
-  const courseGuides = guides.filter(guide => guide.type === 'Course Guide');
-  const genPurposeGuides = guides.filter(guide => guide.type === 'General Purpose Guide');
-  const subjectGuides = guides.filter(guide => guide.type === "Subject Guide")
-  const topicGuides = guides.filter(guide => guide.type === 'Topic Guide');
+  // Group the guides by their type, so we can loop over it later.
+  const groupedGuides = guides.reduce(
+    (entryMap, e) => entryMap.set(e.type, [...entryMap.get(e.type)||[], e]),
+    new Map()
+  );
 
   return (
     <div {...props}>
-      {(courseGuides.length > 0) &&
-        <div className="mb-40">
-          {headingLevel === 2 && <h2 className="type-1">Course Guides</h2>}
-          {headingLevel === 3 && <h3 className="type-1">Course Guides</h3>}
-          <LibGuideSection heading="Course Guides" guides={courseGuides}/>
+      {[...groupedGuides.keys()].map(guideTopic => (
+        <div key={guideTopic}>
```

Comment:
Add `className="mb-40"`

Reply:
Good catch. I just added `space-y-40` to the parent via the node page component. See: https://github.com/SU-SWS/sulgryphon-nextjs/pull/119/files#diff-559377094eac85f2f0ebd97e64e9d47039be55d4ca58213b9db50a79c8347adeR67

Basically it's the same, but doesn't add bottom margin to the last group.

Reply:
ahh yeah, it basically results in the same outcome either way.

Reply:
I suspect the `space-y-40` is safer. But I didn't read anything the tailwind docs that confirm that.

Reply:
There's at least 4 ways to do the same thing. `mb-40` or `pb-40` (not preferred) on the item, `space-y-40` or `flex flex-col gap-40` on the parent. There's probably some others too.

Reply:
> Yeah, I thought about that, but I noticed on my local that for some reason the last mb-40 isn't applied. It defaults to the mb-50 parent div. But this is good info.

Might be [margin collapse](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_box_model/Mastering_margin_collapsing). Now I'm off to figure out when I got subscribed to all the changes in this repo. 😆

Reply:
:rofl: Yeah that must be it. Only the best people are subscribed to this repo :smile:

Reply:
LOL! 

Well, I appreciate your insight anyways, Eric.

Reply:
I love margin collapse. I feel it's underutilized and is the main difference between margin and padding. Now I'll see myself out 🚶‍♂️

Reply:
I guess this thread is going to more people. lol! That's cool. I mean, I'm actually glad because I learn something today.